tests/testsuite.src/used_binaries.at

```diff
@@ -1000,5 +1000,14 @@ AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-show-caret -Wno-others
  > <EOF>
 ]])
 
+
+AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-absolute-paths -Wall prog.cob 2> compiler.output], [1])
+
+AT_CHECK([sed -e "s|$PWD|HOME|" compiler.output], [0],
+[HOME/prog.cob:7: error: CRUD.CPY: No such file or directory
```

Comment:
Instead of escaping the <> please use [[ ]] to check stdout and $SED instead of sed

Reply:
There is no need to escape anymore with $COMPILE :-)